packages/khalti/example/lib/main.dart

```diff
@@ -227,8 +227,7 @@ class _BankingState extends State<Banking> with AutomaticKeepAliveClientMixin {
                       paymentType: widget.paymentType,
                       returnUrl: 'https://khalti.com',
                     );
-
-                    url_launcher.launchUrl(Uri.parse(url));
+                    url_launcher.launch(url);
```

Comment:
Isn't launch deprecated ?

Reply:
`launchUrl` was giving me compile time error. So, I changed it to `launch`.

Reply:
The issue was `url_launcher` plugin was not updated in the `pubspec.yaml` in the example. Updating it now resolved the issue. `launchUrl` is now being used.

Reply:
Try running `flutter pub upgrade` in the example directory.